integtest.sh

```diff
@@ -86,4 +86,4 @@ else
    echo "run security disabled tests"
 fi
 
-npx cypress run --env SECURITY_ENABLED=$SECURITY_ENABLED --spec "cypress/integration/core-opensearch-dashboards/opensearch-dashboards/*.js,cypress/integration/plugins/*/*"
\ No newline at end of file
+TZ=America/Los_Angeles npx cypress run --env SECURITY_ENABLED=$SECURITY_ENABLED --spec "cypress/integration/core-opensearch-dashboards/opensearch-dashboards/*.js,cypress/integration/plugins/*/*"
```

Comment:
why is TZ(timezone ?) needed?

Reply:
to be consistent with developer's env. some tests that verify time string will fail since most servers are on UTC while tests were developed in PDT